client/marketing/overview/welcome-card/index.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import { Component } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { get } from 'lodash';
+import { Button } from '@wordpress/components';
+import Gridicon from 'gridicons';
+import { compose } from '@wordpress/compose';
+import { withDispatch } from '@wordpress/data';
+
+/**
+ * WooCommerce dependencies
+ */
+import { Card } from '@woocommerce/components';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { recordEvent } from 'lib/tracks';
+import withSelect from 'wc-api/with-select';
+import WelcomeImage from './images/welcome.png';
+
+class WelcomeCard extends Component {
+	constructor( props ) {
+		super( props );
+
+		this.hide = this.hide.bind( this );
+	}
+
+	hide() {
+		this.props.updateOptions( {
+			woocommerce_marketing_overview_welcome_hidden: 'yes',
+		} );
+
+		recordEvent( 'marketing_intro_close', {} );
+	}
+
+	render() {
+		const { isHidden } = this.props;
+
+		if ( isHidden ) {
+			return null;
+		}
```

Comment:
If you're only destructuring one property from the props object, then might as well use it directly here?